techmatters/serverless
# Added support for injecting twitter related SSM parameter values into…

Primary reviewer: @GPaoloni 

## Description
* Added support for injecting twitter related SSM parameter values into serverless deployments. 
* Added relevant twitter SSM parameters to ZM prod & staging workflows

TODO: Add Instagram variables

### Checklist
- [ X ] Corresponding issue has been opened
- [ N/A ] New tests added
- [ N/A ] Strings are localized

### Related Issues
Fixes CHI-1060

### Verification steps

Deploy ZM Staging
Check that the twitter environment variables are set correctly for zambia staging in the service in Twilio Console
